src/containers/FactoryLoader.tsx

```diff
@@ -143,6 +141,21 @@ export class FactoryLoaderContainer extends React.PureComponent<Props, State> {
     }
   }
 
+  private async openIde(): Promise<void> {
+    const { history, workspace } = this.props;
+    if (!workspace || WorkspaceStatus[workspace.status] !== WorkspaceStatus.RUNNING) {
+      return;
+    }
+    this.setState({ currentStep: LoadFactorySteps.OPEN_IDE });
+    try {
+      await this.props.requestWorkspace(workspace.id);
+    } catch (e) {
+      this.showAlert(`Getting workspace detail data failed. ${e}`);
+    }
+    await delay();
+    history.push(`/ide/${workspace.namespace}/${workspace.devfile.metadata.name}`);
```

Comment:
We have the helper functions that provide the correct paths for many routes in the application as well as for IDE to open a workspace:`buildIdeLoaderPath`

```suggestion
    history.push(buildIdeLoaderPath(workspace));
```

Reply:
Ok. I have fixed it.